Xamarin.Forms.Core/ImageElement.cs

```diff
@@ -131,16 +126,15 @@ async static void CancelOldValue(ImageSource oldvalue)
 			}
 		}
 
-		public static void ImageSourceChanged(BindableObject bindable, ImageSource newSource)
+		static void ImageSourceChanged(BindableObject bindable, ImageSource newSource)
```

Comment:
LOOK HERE. this is the change that matters. Everything else is cosmetic. DO NOT GET DISTRACTED.